client/src/app/components/CreateProject/Form.js

```diff
@@ -36,17 +36,37 @@ class Form extends React.Component {
 
   onSubmit(e) {
     e.preventDefault();
-    console.log(this.state);
+
+    fetch("http://localhost:3000/projects/createProject", {
+      method: "POST",
+      body: JSON.stringify(this.state),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    })
+      .then(response => {
+        if (response.ok) {
+          response
+            .json()
```

Comment:
I think you need to apply json() in a separate `.then`, before any other `.then`